Engineering/Software-Engineering/L4-Lead-Software-Engineer.md

```diff
@@ -11,84 +11,85 @@
-- I am comfortable with challenging and coaching my peers.
+- I define, document, and evolve best practices in my team's area of focus.
+- I help guide and support the development of new projects across the organisation.
+- I work to expand the technical skill sets of my team.
+- I challenge and coach my peers to improve and grow.
 - I help people determine the most valuable ways they can contribute, and grow, and I help them block out the noise.
+- I help teammates prioritise their efforts, develop their strengths, and stay focused on impactful work.
 
 <details>
 <summary>Examples</summary>
 
-- I provided training and mentoring for multiple team members, deliberately helping them to round out their skill sets.
-- I shared my on-the-job learning and experiences with others so they can understand and be more effective in their own roles.
-- I broke up a project in such a way that lined up appropriate challenges for each of my teammates that helped them grow.
-- I had some difficult conversations with my teammates, challenging them directly while showing them my care for them personally.
-- I adjusted my interactions to adapt to the different working styles and needs of individuals on my team.
+- I trained and mentored team members, helping them develop well-rounded skill sets.
+- I shared my experiences to help others become more effective in their roles.
+- I divided a project into tasks that presented appropriate challenges to help teammates grow.
+- I had constructive conversations that provided direct feedback while showing personal care for teammates.
+- I tailored my interactions to suit the working styles and needs of individual team members.
 
 </details>
 
 ## 🧭 Culture and Leadership
 
 - I encompass all of the qualities listed in [Leadership](https://github.com/OctopusDeploy/People/blob/main/Leadership.md).
-- I avoid becoming a bottleneck, by setting up other engineers on my team for success for longer periods of time before needing to check back in.
+- I empower engineers to work independently and avoid becoming a bottleneck.
 - I make sure difficult work that needs to be done gets done, like hard debugging, incident response, and reverse-engineering tasks.
-- I understand our organisation structure and how we schedule work, and help my team find their highest value contributions in this environment.
-- I am comfortable with transparently assessing risk, making recommendations, escalating appropriately, and dealing with the consequences along the way.
-- I recognise problems and proactively engage with other people to prevent or resolve them quickly.
-- I am comfortable coordinating projects where success requires the input of multiple teams and diverse technical roles.
+- I guide my team to focus on their highest value contributions within the organisation’s structure and scheduling.
+- I assess risk transparently, provide actionable recommendations, escalate when necessary, and manage outcomes responsibly.
```

Comment:
We probably don't even need "responsibly" here. I also think that "transparently" should apply to the whole sentence, not just for assessing risks.

Proposal:
> I transparently assess risk, provide actionable recommendations, escalate when necessary, and manage outcomes.